package/base-files/files/lib/preinit/10_indicate_preinit

```diff
@@ -96,8 +96,13 @@ preinit_config_board() {
 			json_select "$netdev"
 				json_get_vars path path
 				next_eth="$(echo "$netdev" | grep 'eth[0-9]*' | tr -dc '[0-9]')"
-				[ "$next_eth" -gt "$max_eth" ] && max_eth=$next_eth
-				if [ -n "$path" -a -h "/sys/class/net/$netdev" ]; then
+				if [ -n "${next_eth:-}" ] && \
+				   [ "${next_eth}" -gt "${max_eth}" ]; then
```

Comment:
@jow- points out that:

> If you look for all existin eth*s on a system, and take the highest index
> and then you iterate a set of devices and rename to eth${highest_index+n}
> [then] it is guaranteed that there will be no conflict

Perhaps you can skip this check entirely?

Reply:
This is really something I would like to get from the maintainer/author of this code  (e.g. you). You wrote this, so why did you add this check? I don't mind removing it, but I hadn't dug into why you've added this functionality at all :)

So I'm happy to rephrase my MR to only do that little bit of cleanup we discussed priror :p